### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by yatil** [guibbs/w3c-tpac2016#35]
The web app is not accessible at all. 
- Opening the menu is not setting the focus appropriately
- Expand Collapse buttons are not labelled
- Menu button is not labelled
- The map has no text alternative/long description
